src/views/Game/AIReview.tsx

```diff
@@ -952,13 +966,35 @@ export class AIReview extends React.Component<AIReviewProperties, AIReviewState>
             </div>
         );
     }
-    public renderWorstMoveList(): JSX.Element {
-        if (!this.props.game.goban?.engine?.move_tree || !this.ai_review) {
+
+    public getWorstMoveList(): AIReviewWorstMoveEntry[] {
+        const move_tree = this.props.game.goban.engine.move_tree;
+        let worst_moves: AIReviewWorstMoveEntry[];
+        let threshhold: number;
+
+        if (this.ai_review?.scores) {
+            worst_moves = computeWorstMoves(move_tree, this.ai_review, /*use_score=*/true);
+            threshhold = -5.0;
+        } else {
+            worst_moves = computeWorstMoves(move_tree, this.ai_review);
+            threshhold = -0.2;
+        }
+
+        const filtered_worst_moves = worst_moves.filter(de => de.delta <= -5.0);
```

Comment:
Instead of `-5.0` should that be `threshold`?

Reply:
Oops yeah, thanks for catching!